src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -382,16 +382,28 @@
 
 helps['acr build-task logs'] = """
     type: command
-    short-summary: Show logs for a particular build. If no build-id is supplied, it shows logs for the last, or currently executing build.
+    short-summary: Show logs for a particular build. If no build-id is supplied, it shows logs for the last updated build.
     examples:
-        - name: Show logs for the most current, or executing build.
+        - name: Show logs for the last updated build in the registry.
           text: >
             az acr build-task logs -r MyRegistry
+        - name: Show logs for the last updated build in the registry, filtered by build task.
+          text: >
+            az acr build-task logs -r MyRegistry -n MyBuildTask
         - name: Show logs for a particular build.
           text: >
             az acr build-task logs -r MyRegistry --build-id buildId
 """
 
+helps['acr build-task cancel'] = """
+    type: command
+    short-summary: Send a cancellation request for a particular build. If no build-id is supplied, it sends a cancellation request for the last updated build.
```

Comment:
`If build ID is not supplied`.

Reply:
Also see the other comment, I wonder if we want to support this canceling last build.

Reply:
Removed last build support.